apim/3.x/tests/api/configmap_metrics_test.yaml

```diff
@@ -0,0 +1,42 @@
+suite: Test Management API default configmap - Metrics
+templates:
+  - "api/api-configmap.yaml"
+tests:
+  - it: Enable metrics service and set `labels`
```

Comment:
Maybe we could add a test case for the default config (without setting anything just checking the values.yaml)

Reply:
Good catch, just added!